src/Turbo/src/TurboResponse.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Turbo;
+
+use Symfony\Component\HttpFoundation\Response;
+
+class TurboResponse extends Response
+{
+    public function __construct(?string $content = '', int $status = 200, array $headers = [])
+    {
```

Comment:
Both HttpFoundation StreamedResponse and [StreamedJsonResponse](https://github.com/symfony/symfony/blob/f42d8b4e9b3bd4724c24b309dbdbdc11c6e9244d/src/Symfony/Component/HttpFoundation/StreamedJsonResponse.php#L55-L60) have specialized constructors.

What if we made TurboStream a stringable object and have this kind of signature 

```php
/**
 * @param string|Stringable|iterable<string|Stringable> $stream
 */ 
public function __construct(string|iterable $streams = '', int $status = 200, array $headers = [])
{
```

(Genuine suggestion, not sure what would be the best DX here)

Reply:
That's not possible: a response has to extend StreamedResponse if it streams, and not otherwise.
We'd need to have two kind of responses: StreamedTurboStreamResponse and TurboStreamResponse.

BUT, I don't see the need for a StreamedTurboStreamResponse honestly. People should use StreamedResponse with an iterable that uses `TurboStream::*()` instead.